src/WebJobs.Script.WebHost/Models/HttpOperationInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.ObjectModel;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Models
+{
+    [JsonObject]
+    public class HttpOperationInfo
+    {
+        [JsonProperty(PropertyName = "operationId")]
+        public string OperationId { get; set; }
+
+        [JsonProperty(PropertyName = "produces")]
+        public Collection<string> Produces { get; } = new Collection<string> { };
+
+        [JsonProperty(PropertyName = "consumes")]
+        public Collection<string> Consumes { get; } = new Collection<string> { };
+
+        [JsonProperty(PropertyName = "parameters")]
+        public Collection<HttpOperationInputParameterInfo> InputParameters { get; set; }
+
+        [JsonProperty(PropertyName = "description")]
+        public string Description { get; } = "Replace with Operation Object #http://swagger.io/specification/#operationObject";
+
+        [JsonProperty(PropertyName = "responses")]
+        public static JObject Responses => JObject.Parse("{\"200\":{\"description\":\"Success operation\"}}");
```

Comment:
Rather than parse these here, lets just build the default JObjects in a default ctor?